packages/inferno-animation/src/animationCoordinator.ts

```diff
@@ -12,11 +12,12 @@ const NrofPhases = 6;
 
 let _animationQueue: Function[] = [];
 let _animationActivationQueue: Function[] = [];
-let _nextAnimationFrame;
-let _nextActivateAnimationFrame;
+let _nextAnimationFrame: number;
+let _nextActivateAnimationFrame: number;
+const NOOP = 0;
```

Comment:
probably this could be const enum too, otherwise it becomes variable in the javascript bundle
also initialize the variables _nextAnimationFrame _nextActivateAnimationFrame to hint js engine they are numbers

Reply:
FIXED in  e2bfc55 and 4d3778d11
I realised I can just add length as an enum at the end and it will be handled by TS as actual length. Makes for readable code. Is it still worth converting the NOOP to a const enum?